lib/generators/rodauth/views_generator.rb

```diff
@@ -14,6 +14,10 @@ class ViewsGenerator < ::Rails::Generators::Base
           desc: "Generates views for all Rodauth features",
           default: false
 
+        class_option :css, type: :string,
+          desc: "Generates tailwind css specific views",
+          default: "bootstrap"
```

Comment:
I was thinking it would be nice if the `:css` option defaults to `tailwind` when `tailwindcss-rails` gem is used. I like how in this case Rails' built-in scaffold automatically generates Tailwind-friendly templates, and would like to have the same behaviour for rodauth-rails 🙂

Reply:
I can do the same here.

Q: Would you still want to retain the `class_option`?

I would throw in my two cents on the above question: 

- going forward on rails +7 apps, the canonical approach would probably be to use the [`cssbundling-rails`](https://github.com/rails/cssbundling-rails) gem. i.e. via node, rather than the `tailwindcss-rails` gem.
- Would there be other styles you would want rodauth-rails to perhaps support: e.g. `semantic-ui` etc.

Reply:
Yes, I would prefer to keep the class option, to enable generating Tailwind templates for Rails apps using Tailwind without `tailwindcss-rails` (e.g. via `cssbundling-rails` that you mentioned).

Reply:
[3cc210b](https://github.com/janko/rodauth-rails/pull/114/commits/3cc210b8b8a4c1227ce98f732d8742a9438dce9d)